dataclasses_avroschema/pydantic/parser.py

```diff
@@ -18,6 +18,16 @@ def get_field_metadata(field_info: FieldInfo) -> dict[str, typing.Any]:
         )
         if field_info.description:
             metadata["doc"] = field_info.description  # type: ignore
+
+        aliases = set(metadata.get("aliases", []))
+
+        if field_info.alias:
```

Comment:
Compare `field_info.alias` and `field_info.serialization_alias` using `is not None`

Reply:
This is why I create a `set` to remove any duplicates in case `metadata["aliases"]` could have previously been set, as well as considering `Field.alias` and `Field.serialization_alias`.

Reply:
Yes, that is good but in python you compare against `None`when comparing `n object identity `field_info.alias` is `str | None` then you must use `if field_info.alias is not None: ..`. The same for `field_info.serialization_alias`

Reply:
Not testing against explicitly against `None` means that empty string aliases are ignored.  Are you sure the alternative is desirable behavior?  (p.s. I've used Python for over 30 years-- I'm quite familiar with the semantics.)

Reply:
That is a good question. It is possible to use `""` as alias, of course it does not make sense. So, let's go with your solution!